test/GitcoinGovernor.t.sol

```diff
@@ -11,35 +11,67 @@ import {IGovernorAlpha} from "src/interfaces/IGovernorAlpha.sol";
 import {IGTC} from "src/interfaces/IGTC.sol";
 import {ProposeScript} from "script/Propose.s.sol";
 
-contract GitcoinGovernorTestHelper is Test, DeployInput {
+abstract contract GitcoinGovernorTestHelper is Test, DeployInput {
   using FixedPointMathLib for uint256;
 
   uint256 constant QUORUM = 2_500_000e18;
   address constant GTC_TOKEN = 0xDe30da39c46104798bB5aA3fe8B9e0e1F348163F;
+  IGTC gtcToken = IGTC(GTC_TOKEN);
   address constant TIMELOCK = 0x57a8865cfB1eCEf7253c27da6B4BC3dAEE5Be518;
   address constant PROPOSER = 0xc2E2B715d9e302947Ec7e312fd2384b5a1296099; // kbw.eth
-  address[] delegates = [
-    PROPOSER, // kbw.eth (~1.8M)
-    0x2df9a188fBE231B0DC36D14AcEb65dEFbB049479, // janineleger.eth (~1.53M)
-    0x4Be88f63f919324210ea3A2cCAD4ff0734425F91, // kevinolsen.eth (~1.35M)
-    0x34aA3F359A9D614239015126635CE7732c18fDF3, // Austin Griffith (~1.05M)
-    0x7E052Ef7B4bB7E5A45F331128AFadB1E589deaF1, // Kris Is (~1.05M)
-    0x5e349eca2dc61aBCd9dD99Ce94d04136151a09Ee // Linda Xie (~1.02M)
-  ];
+  address constant DEPLOYED_BRAVO_GOVERNOR = 0x1a84384e1f1b12D53E60C8C528178dC87767b488;
+
+  struct Delegate {
+    string handle;
+    address addr;
+    uint96 votes;
+  }
+
+  Delegate[] delegates;
 
   GitcoinGovernor governorBravo;
 
   function setUp() public virtual {
-    uint256 _forkBlock = 15_980_096; // The latest block when this test was written
+    // The latest block when this test was written. If you update the fork block
+    // make sure to also update the top 6 delegates below.
+    uint256 _forkBlock = 17_032_826;
     vm.createSelectFork(vm.rpcUrl("mainnet"), _forkBlock);
 
-    DeployScript _deployScript = new DeployScript();
-    _deployScript.setUp();
-    governorBravo = _deployScript.run();
+    // Taken from https://www.tally.xyz/gov/gitcoin/delegates?sort=voting_power_desc.
+    // If you update these delegates (including updating order in the array),
+    // make sure to update any tests that reference specific delegates.
+    Delegate[] memory _delegates = new Delegate[](6);
+    _delegates[0] = Delegate("kevinolsen.eth", 0x4Be88f63f919324210ea3A2cCAD4ff0734425F91, 1.8e6);
```

Comment:
What's the point of hardcoding the voting weights here if we just end up fetching them again below? I.e. why not set them to 0?

Reply:
Just for our own benefit. They were initially comments (as you know) and I felt the comments were still valuable to have -- i.e. we don't want to have to manually look up voting weights while we're writing tests, we want to just see ballpark numbers to figure out who to have vote for what.

And it just seemed silly to have the comments and not just use the dedicated field